From 821b7f6ad32c005aec731bdd9d1c8f0c8fe29326 Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@open-nandra.com>
Date: Mon, 24 May 2021 12:40:59 +0200
Subject: [PATCH] sunxi-common: Use for arm64 same path as for arm

Without this fdtfile is set with prefix 'allwinner' and boot.scr cannot
find devicetree.

Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>
---
 include/configs/sunxi-common.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/configs/sunxi-common.h b/include/configs/sunxi-common.h
index f4388ea5fa..632cca737e 100644
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -382,11 +382,7 @@
 	CONSOLE_STDIN_SETTINGS \
 	CONSOLE_STDOUT_SETTINGS
 
-#ifdef CONFIG_ARM64
-#define FDTFILE "allwinner/" CONFIG_DEFAULT_DEVICE_TREE ".dtb"
-#else
 #define FDTFILE CONFIG_DEFAULT_DEVICE_TREE ".dtb"
-#endif
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	CONSOLE_ENV_SETTINGS \
-- 
2.25.1

